### PR TITLE
Rewind request before reading [##114350111]

### PR DESCRIPTION
**Pull request by NC-piercej**
`rake release` after this is merged, correct?

### CODE REVIEW

File: lib/rack/request_logger.rb [hunk 1]
```diff
@@ -102,6 +102,7 @@ def log_handler
 
     def read_stream(data)
       return nil unless data
+      data.rewind
```
Comment (marionm) [inline lib/rack/request_logger.rb]: Test update?

File: lib/rack/request_logger_version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module Rack
   class RequestLogger
-    VERSION = '0.0.5'
+    VERSION = '0.0.6'
```
Comment (marionm) [inline lib/rack/request_logger_version.rb]: For gems, keep the version bumps out of PRs, and bump it directly on master.

Don't really have to change it for this PR, it's just a cleaner convention for the future. Doesn't matter much for these super low traffic gems, of course.

**Comment by marionm** [comment]
👍 after test update.

Correct about `rake release`. Some people have had trouble with it though, so if that doesn't work, check out `geminabox`.

File: lib/rack/request_logger_version.rb (hunk 2 above)
Comment (pittmanw) [inline lib/rack/request_logger_version.rb]: I generally like to include the version bump in the PR while noting the commit should not be merged. (e.g. `DO NOT MERGE Bump version to 0.0.6`) Once the PR has been reviewed, the commit can be stripped off and applied to master after the PR is merged.

I think it's useful to let PR reviewers sanity check the version bump to make sure we're following our conventions.

Comment (marionm) [inline lib/rack/request_logger_version.rb]: That makes sense too.

**Pull request merged**